modules/oadp-installing-dpa.adoc

```diff
@@ -172,28 +172,29 @@ spec:
         provider: {provider}
         default: true
         credential:
-          key: cloud
-          name: {credentials} <5>
+          key: cloud <5>
+          name: {credentials} <6>
         objectStorage:
-          bucket: <bucket_name> <6>
-          prefix: <prefix> <7>
-  snapshotLocations: <8>
+          bucket: <bucket_name> <7>
+          prefix: <prefix> <8>
+  snapshotLocations: <9>
     - velero:
         provider: {provider}
         default: true
         config:
           project: <project>
-          snapshotLocation: us-west1 <9>
+          snapshotLocation: us-west1 <10>
 ----
 <1> The `openshift` plugin is mandatory.
 <2> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
 <3> Set to `false`, if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You can configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
 <4> Specify on which nodes Restic is available. By default, Restic runs on all nodes.
-<5> If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
-<6> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
-<7> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
-<8> Specify a snapshot location, unless you use CSI snapshots or Restic to back up PVs.
-<9> The snapshot location must be in the same region as the PVs.
+<5> For Google WIF cloud authentication use the `service_account.json`, which is `cloud-credentials-gcp`.
```

Comment:
```suggestion
<5> Secret key that contain credentials. For Google WIF cloud authentication use the `service_account.json` for key.
```

`which is cloud-credentials-gcp` which is part of <6> which I think is covered already by 
> <6> If you do not specify this value, the default name, `{credentials}`

so I think perhaps we can remove this one

Reply:
IIUC {credentials} for GCP would show up as `cloud-credentials-gcp` based on [this](https://github.com/openshift/openshift-docs/blob/5759fa681dd1cf3ac9262ce7ce88b2aea12dceeb/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc?plain=1#L7)

Reply:
@kaovilai 
You are correct the {credentials} for GCP display as `cloud-credentials-gcp`.

So with and without the Google WIF authentication the DPA stays the same?

Reply:
@kaovilai 

Is this what should be written in <5> key & <6> name:

<5> Secret key that contain credentials.
<6> If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location. For Google WIF cloud authentication use the `service_account.json.`

Reply:
Almost. Sorry. I'll write <5> and <6> here.
```
<5> Secret key that contain credentials. For Google WIF cloud authentication use service_account.json.
<6> Secret name that contains credentials If you do not specify this value, the default name, {credentials}, is used. 
```

Reply:
Thank you!

Reply:
@kaovilai 
Can I send this to QE for review?

Reply:
yes. please send to qe for review.